homeassistant/components/webmin/sensor.py

```diff
@@ -105,7 +105,7 @@ def __init__(
         super().__init__(coordinator)
         self.entity_description = description
         self._attr_device_info = coordinator.device_info
-        self._attr_unique_id = f"{coordinator.mac_address}_{description.key}"
+        self._attr_unique_id = f"{coordinator.unique_id}_{description.key}"
```

Comment:
I suggest using a "-" to split between the unique id and the key in case the key has underscores in it.